.github/workflows/reusable_guidelines_enforcer.yml

```diff
@@ -31,21 +37,22 @@ jobs:
     secrets:
       git_token: ${{ secrets.git_token }}
 
-  call_check_is_rust:
-    name: Check if app is based on Rust
-    uses: ./.github/workflows/_check_is_rust.yaml
+  call_get_app_metadata:
```

Comment:
This is annoying that we have to define this multiple times, but there is no way to depend on jobs from another  workflow right ?

Reply:
What do you mean by "multiple times"?

I _think_ (not sure) it is possible to link workflows, but I guess that would greatly complexify sharing variables (to bind workflow outputs to another job's inputs)

Reply:
Let's drop for now